packages/design-system/src/components/cookieDetails/details.tsx

```diff
@@ -100,19 +100,17 @@ const Details = ({ selectedCookie, isUsingCDP }: DetailsProps) => {
           </p>
         </div>
       )}
-      {(outboundBlock || inboundBlock) &&
-        hasValidBlockedReason &&
-        isUsingCDP && (
-          <>
-            <p className="font-bold text-raising-black dark:text-bright-gray mb-1">
-              Blocked Reason
-            </p>
-            <p
-              className="text-outer-space-crayola dark:text-bright-gray"
-              dangerouslySetInnerHTML={{ __html: blockedReasons ?? '' }}
-            />
-          </>
-        )}
+      {hasValidBlockedReason && isUsingCDP && (
+        <>
+          <p className="font-bold text-raising-black dark:text-bright-gray mb-1">
+            Blocked Reason
+          </p>
+          <p
+            className="text-outer-space-crayola dark:text-bright-gray"
+            dangerouslySetInnerHTML={{ __html: blockedReasons ?? '' }}
+          />
+        </>
+      )}
```

Comment:
@amovar18 I remember having a discussion about this. Would you mind adding  comments about this in your PR description.

Reply:
This was to make sure the blocking reason being displayed follows the same logic that is used to display the blocking status.